components/automate-cli/cmd/chef-automate/diagnostics.go

```diff
@@ -62,7 +69,10 @@ func runDiagnosticsRunCmd(cmd *cobra.Command, args []string) error {
 
 	dsClient, err := client.Connection(client.DefaultClientTimeout)
 	if err != nil {
-		return err
+		logrus.WithError(err).Warn("could not connect to deployment service")
+		if diagnosticsRunCmdOpts.adminToken == "" {
+			return err
```

Comment:
Is the idea here that we can run a lot of the tests with a hand-created adminToken even if we can't connect to ds so we don't need this client later?  I'm cool with this for now, but as a follow up maybe we can move the adminToken retrieval code a bit closer to this code so that is clear.

Reply:
that sounds right. i did things so i could run the cli locally